pkg/kubectl/resource_printer.go

```diff
@@ -60,6 +60,9 @@ const (
 	loadBalancerWidth = 16
 )
 
+// This indicates that a particular StorageClass nominates itself as the system default.
+const betaAnnotation = "storageclass.beta.kubernetes.io/is-default-class"
```

Comment:
that part has been all sorts of discussed between thockin and Jan. I'm more interested if its worth it to define something `isDefaultAnnotation` on the storage class object or something, instead of having the `const` name twice...